### PR TITLE
Parity upfront

### PR DESCRIPTION
**Pull request by brookwander**
This will take care of half of the parity objectives. Parity plots will be generated at the start of every new run selecting data that is representative of the desired materials + adsorbate. 

ONE THING: mpid filtering has not been implemented. I have not devised an approach that I am satisfied with. If you have thoughts please share. I will be bringing this up at the weekly meeting.

### CODE REVIEW

File: catlas/filters.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def bulk_filter(config, dask_df):
                         meta=("bulk_elements", "bool"),
                     )
                 ]
+
```
Comment (KirbyBroderick) [inline catlas/filters.py:28]: Was this misformatted or something?

**Review by KirbyBroderick — APPROVED** [review]
push once tests pass

**Review by KirbyBroderick — APPROVED** [review]
Looks good. `parity_utils` needs documentation and has a lot of repeated code, though (all the calls to set up plots with identical axes in `get_general_plot` and `get_specific_smile_plot` in particular)

File: bin/predictions.py [hunk 2]
```diff
@@ -20,7 +28,7 @@
     safe_cache,
     better_build_func_identifier,
 )
-
+import warnings
```
[the remark this answers is not in the document]
Comment (brookwander) [reply bin/predictions.py:31]: > Looks good. `parity_utils` needs documentation and has a lot of repeated code, though (all the calls to set up plots with identical axes in `get_general_plot` and `get_specific_smile_plot` in particular)

will be included with the next set of changes, thanks!

**Pull request merged**